### PR TITLE
Improve logging; simplify font handling

### PR DESCRIPTION
**Pull request by sumpfork**
Use loguru for logging. Turn down log level by default but add option to set it.

Clean up font use in the non-latin1 case.

@smrky1

### CODE REVIEW

File: src/domdiv/cards.py [hunk 1]
```diff
@@ -208,11 +209,7 @@ def setImage(self, use_set_icon=False):
                     setImage = Card.sets[self.cardset_tag]["image"]
 
         if setImage is None and self.cardset_tag != "base":
-            print(
-                'warning, no set image for set "{}", card "{}"'.format(
-                    self.cardset, self.name
-                )
-            )
+            logger.warning('no set image for set "{self.cardset}", card "{self.name}"')
```
Comment (nickv2002) [inline src/domdiv/cards.py:212]: Nice upgrade, no need for f-strings I assume.

File: src/domdiv/draw.py [hunk 2]
```diff
@@ -586,100 +586,90 @@ def registerFonts(self):
...
-            pdfmetrics.registerFont(
-                TTFont(
-                    "Times-Italic-TTF",
-                    pkg_resources.resource_filename(
-                        "domdiv", fontpaths.get("Times-Italic-TTF")[0]
-                    ),
-                )
-            )
+        timesTTFs = set(["Times-Roman-TTF", "Times-Bold-TTF", "Times-Italic-TTF"])
+        timesTTF_not_found = timesTTFs - set(fontpaths)
```
Comment (nickv2002) [inline src/domdiv/draw.py:598]: 👍 good use of `set`

**Review by nickv2002 — APPROVED** [review]
Nice upgrades to logging and readability of the code.

File: src/domdiv/cards.py (hunk 1 above)
Comment (sumpfork) [reply src/domdiv/cards.py:212]: Actually no, I think this is wrong :). From the docs I initially thought there was no need, but there is. The lib can do `format`-style substitution but then needs the args supplied explicitly, so I went with standard f-strings instead. I'll fix this.

**Pull request merged**